addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -415,7 +419,12 @@ var KanbanRenderer = BasicRenderer.extend({
 
         // append ghost divs to ensure that all kanban records are left aligned
         var prom = Promise.all(self.defs).then(function () {
-            return self._renderGhostDivs(fragment, 6);
+            var options = {};
+            var $recordView = $(fragment).find(".o_kanban_record:not(.o_kanban_ghost)");
+            if ($recordView.length && $recordView.attr("style")) {
+                options.inlineStyle = $recordView.attr("style");
+            }
+            return self._renderGhostDivs(fragment, 6, options);
```

Comment:
wouldn't it be better to simply remove the 'width: 400px' in the arch, since it isn't correctly applied anyway?

Reply:
What do you mean, it is not correctly applied ? :) Without them, the kanban are small. With them, the kanban are large ;)

Reply:
Yep, but the cards never have a 400px width

Reply:
Actually, trying to force a width is weird, as the width is automatically computed to fit the screen. What we could do (or is it already done?) is have something like sizes attributes or classnames (like s, m or l)

Reply:
Probably better yes, but your classname would set that 400px width anyway (to change the default which is 300 which is never the final size of the cards either). CSS widths are not computed width ;)

Reply:
Isn't it rather min and max widths?

Reply:
Could maybe work with min-width but you would have to set a min-width smaller than the smallest phone width or add media queries and lose the point of using flex. Here the width is used as a flex starting point (as using flex-basis directly was not working in all browsers)... kanban css needs to be refactored... or I would not mess with it :smile:

Reply:
ok you're right, i didn't remember how it worked